### ISSUE TITLE
Settings menu not working

### ISSUE THREAD

**Issue by albertotn** [mirkoperillo/resting#135]
## Expected Behavior
When I click on settings menu, the menu is displayed

## Actual Behavior
No menu is displayed

## Steps to Reproduce the Problem

  1. install from scratch Resting on new firefox
  1. click on settings menu

## Specifications

  - Addon Version: 1.2.2
  - Firefox Version: 95.0.2
  - OS: Windows 10

## Notes

There is an error in the console:
```

Uncaught Error: Bootstrap's JavaScript requires jQuery
    <anonymous> moz-extension://408b8366-32a2-43e5-b28e-a3c54d19c6f4/js/vendor/bootstrap.js:8
```

**Comment by mirkoperillo** [comment]
Sometime Firefox doesn't load correctly the jQuery library.
Try to reload extension with `F5`, normally this workaround works.

Let me know if the suggestion resolves the problem

**Comment by albertotn** [comment]
No it doesn't work ( tested on both work and personal pc same setup and versions )

**Comment by mirkoperillo** [comment]
I'm not able to reproduce your problem.
My experience is that after a addon reload, the problem is resolved.

This is my experience on a Linux machine, I'm going to try it on Windows as soon as I'll have the possibility.

**Comment by mirkoperillo** [comment]
@albertotn I haved tested the problem on a Windows machine and yes, I can confirm it.
It seems Firefox on Windows has a different mechanism to load the web-extension dependencies.
Tested on:
* Windows 10
* Firefox 96.0.3
* Resting 1.3.0

I'm going to fix it asap.

**Comment by mirkoperillo** [comment]
@albertotn I had just released a fix version for this issue, v1.3.1

I tested on Windows and everything seems alright.
Feel free to reopen the issue if you have other problems about

**Comment by albertotn** [comment]
works just fine, thanks!
